app/samples/tftp_server/include/c_publisher.h

```diff
@@ -0,0 +1,40 @@
+#ifndef PRINTCOUNT_H
```

Comment:
print count?

Reply:
OH LMAO. Whenever I make a new sample I copypasta another sample and must've missed changing these

Reply:
Actually these shouldnt even be part of the sample